test/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanTest.java

```diff
@@ -0,0 +1,48 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static org.junit.Assert.assertEquals;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class DynamoSchedulePlanTest {
```

Comment:
It won't pass because the class is not final. I think DDB prevents us from making it final.... I'll test that.